Games/types/Mafia/Action.js

```diff
@@ -373,4 +373,35 @@ module.exports = class MafiaAction extends Action {
     const rightIdx = (index + 1) % alive.length;
     return [alive[leftIdx], alive[rightIdx]];
   }
+  
+  getVanilla(player) {
+    player = player || this.target;
+    var vanillaRole;
+    if (player.role.alignment === "Village" || player.role.winCount === "Village") {
```

Comment:
winCount Village is mostly deprecated, it is synonymous with "Independent"

should a role like Survivor turn to Villager or Grouch?

How about the Vanilla role for Independent being Survivor instead?